aarch64/latest.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE HTML>
```

Comment:
We shouldn't need this anymore so we should be able to remove this (along with the others) in a separate PR.